packaging/post_build_script_linux.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+pip3 install auditwheel pkginfo
+
+for pkg in dist/torchdata*.whl; do
+    echo "PkgInfo of $pkg:"
+    pkginfo $pkg
+
+    auditwheel show $pkg
```

Comment:
I'm sorry that I forget to say you have to add
```
auditwheel repair $pkg --plat manylinux2014_x86_64
```
By default new manylinux-compatible wheels are stored under `wheelhouse/` unless `-w <NEW_WHEEL_DIR>` is specified.

I am not sure how to do that. But, we need to upload new wheels from `wheelhouse` rather than from `dist`.